contents/handbook/engineering/oncall.md

```diff
@@ -12,14 +12,20 @@ Make sure you've downloaded the PagerDuty app for your phone, and that you've se
 
 ## What to do when you get paged
 
-A chunk of our high-frequency alerts have runbooks attached, most of which live [in the runbooks section](https://posthog.com/docs/runbook) of our docs. The runbook should tell you what to look at and easy fixes if there are any. Every alert also has a link to the grafana graph that triggered the alert.
+A chunk of our high-frequency alerts have runbooks attached, most of which live [in the public runbooks section](https://posthog.com/docs/runbook) of our docs or in [our private runbooks](http://go/runbooks). The runbook should tell you what to look at and easy fixes if there are any. Every alert also has a link to the grafana graph that triggered the alert.
 
 If this looks like the page [should be raised as an incident](https://posthog.com/handbook/engineering/incidents#when-to-raise-an-incident), [go raise an incident](https://posthog.com/handbook/engineering/incidents).
 
-If you are unsure how to proceed, get the [secondary for the relevant team](https://posthog.com/handbook/engineering/support-hero#2-secondary-on-call) to shadow you while you figure out how to resolve it. The idea is that they can help you understand the issue and where to find how to debug it. The idea is _not_ for them to take over at this point, as otherwise you won't be able to learn from this page.
+If you are unsure how to proceed, escalate to the [secondary for the relevant team](https://posthog.com/handbook/engineering/support-hero#2-secondary-on-call).
+
+They will either resolve the issue or shadow you while you figure out how to resolve it. The ideal is for them _not_ to take over at this point, as otherwise you won't be able to learn from this page. But, as we grow, we can't all have deep expertise in every part of the system, and sometimes it makes sense to split the work needed to resolve the alert.
+
+Regardless, you as the original person paged are responsible for administration of the page/incident. Tasks like keeping our external status page updated, organising and writing the post incident review, and updating the runbook.
```

Comment:
imho it makes more sense for the product who's users were impacted to write the post mortem (they will better understand what should be happening and if something should be changed about the product, e.g. reduce dependencies, which the on-call would be a poor person to make decisions on) & update the runbook, but the on-call is responsible for making sure it does happen.

Furthermore any pages happening in the US west timezone we should help with that load - as in we can't expect jams to get anything done if he needs to run all the postmortems and update runbooks.

Reply:
Plus it can be better for the post-mortem writer not to have the deep knowledge as they'll have to translate it out which should help make sure everyone can understand/learn

Reply:
And the "admin" of the post-mortem is probably anyway

```
"We did {one thing} and {another thing}" @the-person-that-actually-understood-the-fix is that right?
```

Reply:
Yeah, I disagree

Customers impacted isn't a valuable metric here

Imagine the following:

* someone from product analytics is on call
* on day 1 ClickHouse can't write to `session_replay_events` (for some imaginary reason)
* this causes ingestion lag 
* product analytics person escalates to pipeline because its ingestion
* they together diagnose its ClickHouse
* Jams wakes up and does some black magic and swearing and fixes it
* they ping the session recording team - "Hey your customers were impacted can you write the post-mortem here"

The product team of impacted customers in this example are writing a post-mortem for an incident they weren't involved in.

You could make an argument for at least three different teams being responsible here.

The simplest outcome is "person who was paged or who was incident lead takes the responsibility for making sure it happens". That might mean writing it or might mean hassling other people to write it.

Reply:
Flip it around to help me understand...

Why do you think the product team should write the post-mortem? 

We might be optimising for different things

Reply:
Here's a second example to tease out the solution

* someone from growth team is on call
* ClickHouse can't write to any tables (for some imaginary reason)
this causes ingestion lag
* growth person escalates to pipeline because its ingestion
* they together diagnose its ClickHouse
* Jams wakes up and does some black magic and swearing and fixes it

Because all ingestion was delayed the impacted customers are product analytics, feature success, and session recordings.

Which person is responsible for writing the post-mortem?